css/dialog.css

```diff
@@ -0,0 +1,75 @@
+.item-dialog {
+    width: 95%;
+    margin: auto auto;
+    background-color: var(--primary-light);
+    border: none;
+    padding: var(--size-4);
+}
+
+.item-dialog::backdrop {
+    background-color: rgba(0, 0, 0, 0.301);
+}
+
+.item-dialog-form {
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.item-dialog-form h3 {
+    margin: var(--size-3) 0;
+}
+
+.item-dialog-form label {
+    font-size: var(--font-size-small);
+}
+
+.item-dialog-form input {
+    padding-left: var(--size-1);
+}
+
+.item-dialog-form input,
+.item-dialog-form select {
+    width: 100%;
+    height: var(--size-4);
+    margin-top: var(--size-1);
+    margin-bottom: var(--size-4);
+    font-size: var(--font-size-small);
+}
+
+.item-dialog-form button {
+    padding: var(--size-2);
+}
+
+.item-dialog-btn-container {
+    width: 120px;
+    min-height: var(--size-4);
+    display: flex;
+    justify-content: space-between;
+}
+
+.cancel-btn {
+    background-color: var(--color-delete);
+}
+
+.cancel-btn:hover {
+    color: var(--color-delete);
```

Comment:
A bit weird choice, on hover the text would match the background color? making the text invisible?

Reply:
.cancel-btn also has .button class witch makes the background-color white.

Reply:
consider having the `.button` styles accept a variable. For example:

```
.button {
  background-color: var(--button-bg-color);
  color: var(--button-text-color);
}

.button:hover {
  background-color: var(--button-bg-color-hover);
  color: var(--button-text-color-hover);
}

.cancel-btn {
  --button-text-color: var(--color-delete);
  --button-text-color-hover: var(--color-delete);
}
```

Something like that -- otherwise it is quite hard to know what text color is on the `cancel-btn`. Maybe re-declaring the color on the default state would be even simpler:

```
.cancel-btn {
  color: ...;
}

.cancel-btn:hover {
  color: ...; /* more clear on what's being overriden */
}
```